orca-queue-tck/src/main/kotlin/com/netflix/spinnaker/orca/q/ExecutionLatch.kt

```diff
@@ -43,7 +43,7 @@ class ExecutionLatch(private val predicate: Predicate<ExecutionComplete>) :
     }
   }
 
-  fun await() = latch.await(5, TimeUnit.SECONDS)
+  fun await() = latch.await(10, TimeUnit.SECONDS)
```

Comment:
seems fine... 
but.. @asher : curious why would this be slower now that it's in SQL? is it because we spin up a docker for it? If that's the case maybe whatever spins it up can block so we don't play this cat/mouse game?

Reply:
This isn't related to docker startup times and test execution does block on container db availability. Wouldn't have even been able to persist the pipeline earlier in the test otherwise, and the you'd see the first `QueueIntegrationTest` fail if that was related. Not sure if this PR is the right answer though.